src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -22,28 +22,47 @@ const MAC_REGEX = /^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})|([0-9a-fA-F]{4}\\.[0
 const BMC_REGEX = /^((ipmi|ibmc(\+https?)?|idrac(\+https?)?|idrac-redfish(\+https?)?|idrac-virtualmedia(\+https?)?|irmc|redfish(\+https?)?|redfish-virtualmedia(\+https?)?|ilo4(\+https)?|ilo4-virtuallmedia(\+https)?|ilo5(\+https)?|ilo5-redfish(\+https)|ilo5-virtualmedia(\+https)|https?|ftp):(\/\/([a-z0-9\-._~%!$&'()*+,;=]+@)?([a-z0-9\-._~%]+|\[[a-f0-9:.]+\]|\[v[a-f0-9][a-z0-9\-._~%!$&'()*+,;=:]+\])(:[0-9]+)?(\/[a-z0-9\-._~%!$&'()*+,;=:@]+)*\/?|(\/?[a-z0-9\-._~%!$&'()*+,;=:@]+(\/[a-z0-9\-._~%!$&'()*+,;=:@]+)*\/?)?)|([a-z0-9\-._~%!$&'()*+,;=@]+(\/[a-z0-9\-._~%!$&'()*+,;=:@]+)*\/?|(\/[a-z0-9\-._~%!$&'()*+,;=:@]+)+\/?))(\?[a-z0-9\-._~%!$&'()*+,;=:@/?]*)?(#[a-z0-9\-._~%!$&'()*+,;=:@/?]*)?$/i;
 const LOCATION_CHARS_REGEX = /^[a-zA-Z0-9-._]*$/;
 
+export const CLUSTER_NAME_VALIDATION_MESSAGES = {
+  INVALID_LENGTH: '1-54 characters',
+  INVALID_VALUE: 'Use alphanumberic characters, or hyphen (-)',
+  INVALID_START_END: 'Must start with a letter and end with a letter or number.',
+};
+
+export const UNIQUE_CLUSTER_NAME_VALIDATION_MESSAGES = {
```

Comment:
```suggestion
export const uniqueClusterNameValidationMessages = {
```

Reply:
I would even suggest moving these messages to a different file...

Reply:
I was following a "convetion" we had with the other validation messages...
But I do agree, moving all of them to a different file might be a good idea.